tests/conftest.py

```diff
@@ -81,13 +84,13 @@ async def asgi_app(scope: Scope, receive: Receive, send: Send) -> None: ...
 
 
 @pytest.fixture()
-def redis_store(redis_client: AsyncRedis) -> RedisStore:
-    return RedisStore(redis=redis_client)
+def redis_store(redis_client: Redis) -> RedisStore:
+    return RedisStore(redis=redis_client, handle_client_shutdown=True)
```

Comment:
we need to explicitely handle shutdown in test or we're left with dangling ressources